marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/artifacts/getArtifactsWithProjectPaths.sjs

```diff
@@ -30,8 +30,7 @@ const userArtifactQuery = cts.andQuery([
     // cannot be managed via HC
     consts.FLOW_COLLECTION,
     consts.ENTITY_MODEL_COLLECTION,
-    "http://marklogic.com/data-hub/steps/ingestion",
-    "http://marklogic.com/data-hub/steps/mapping"
+    consts.STEP_COLLECTION
```

Comment:
@ryanjdew , @rjrudin 
I think we originally didn't include the steps that can't be created/modified in HC. Since custom steps can't be created/modified in HC, should that be included as well ?

Reply:
@srinathgit  That's correct, that's my mistake. @ryanjdew sorry about that, you had it correct before with each of the non-custom-step collections listed out.

Reply:
I've adjusted it to exclude custom steps.